src/01-functions.js

```diff
@@ -112,7 +128,13 @@ function averageThree() {}
  *  compareTypes("left", 10);
  *  //> false
  */
-function compareTypes() {}
+function compareTypes(a, b) {
+  if (typeof a === typeof b) {
```

Comment:
This will work for this problem, but if you want to give yourself a little bonus I want to challenge you to think of some cases where this would not work and how you would handle that programmatically!